src/generic/Matrix.jl

```diff
@@ -404,6 +404,18 @@ function Base.view(M::AbstractAlgebra.MatElem{T}, rows::Colon, cols::Colon) wher
    return view(M, 1:nrows(M), 1:ncols(M))
 end
 
+Base.firstindex(M::AbstractAlgebra.MatElem, i::Int) = 1
```

Comment:
Couldn't this be made
```suggestion
Base.firstindex(M::AbstractAlgebra.MatrixElem, i::Int) = 1
```
?